jobs/texastribune_software_engineer_05_2018.html

```diff
@@ -0,0 +1,65 @@
+---
+title: Software Engineer
+company: The Texas Tribune
+url: https://www.texastribune.org/
+location: Austin, TX
+contract: permanent
+contact:
+    name: Liam Andrew
+    email: engineering-jobs@texastribune.org
+    phone: +1 512 716 8600
+created: !!timestamp '2018-05-07'
+tags:
+  - Texas
```

Comment:
You don't need to include the location in the tags - that's why we have a location field.